extensions/pinboard/package.json

```diff
@@ -20,6 +20,13 @@
       "subtitle": "Pinboard",
       "description": "Searches your Pinboard bookmarks by tags.",
       "mode": "view"
+    },
+    {
+      "name": "searchBookmarksWithConstantTags",
+      "title": "Search Bookmarks with constant tags",
```

Comment:
One more nit: Command titles should use title-case, so: Search Bookmarks with Constant Tags